web/src/api/models/master/types/master.ts

```diff
@@ -15,7 +15,27 @@
  * limitations under the License.
  */
 
+export interface MasterLeader {
+  hostname: string
+  ip: string
+  state: string
+  rpcPort: number
+  restPort: number
+  version: string
+}
+
 export interface MasterOverview {
   leaderNum: number
   followerNum: number
+  leader: MasterLeader
```

Comment:
```suggestion
  leader: Master
```
Meanwhile, remove interface `MasterLeader`.

Reply:
ok.